src/ts/src/common/utils/urlUtils.tsx

```diff
@@ -1,6 +1,6 @@
-export function stripProtocol(url: string | undefined): string | undefined {
-  if (url === undefined) {
-    return undefined;
+export function stripProtocol(url?: string): string {
+  if (!url) {
+    return "";
   }
   const re = /^.*:\/\//i;
```

Comment:
btw, shouldn't this be `^[^:]*:\/\///i;`?

Reply:
Hmm I'm not sure familiar with REGEX 😆 What's the difference here?

It seems like `^.*` is capturing beginning of a string where 0 or more any char before `://`, and `^[^:]*` is capturing beginning of a string where 0 or more non `:` char before ://?

So:

```ts
':://www.g.com'.replace(/^[^:]*:\/\//i, '') //  --> ":://www.g.com"
':://www.g.com'.replace(/^.*:\/\//i, '') //  --> "www.g.com"
```

Reply:
Technically, the : is not a valid character for any component of the URL, but that does presume that everyone is being a good actor and escaping things correctly.  My proposed regex strips everything up to the first `:`, rather than to the last `:`.

Reply:
ahh I see! What's your thought on stripping everything before `//`? Like: `/^.*\/\//i`

Reply:
That strips everything before the last `//`.  I generally prefer less greedy regexes.

Reply:
Ahh got it! That sounds good 👍 Thanks @ttung ! [PR](https://github.com/chanzuckerberg/aspen/pull/279) out 😄 PTAL thank you!